File: SoloLearn Codes/Snap, Crackle and Pop/SnapCrackleAndPop.java
import java.util.Scanner;

public class SnapCrackleAndPop
{
    public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
        for(int i = 0; i < 6; i++) {
            int bowl = sc.nextInt();
            System.out.print(bowl % 3 == 0 ? "Pop " : bowl % 2 == 0 ? "Crackle " : "Snap ");
        }
	}
}

File: SoloLearn Codes/Snap, Crackle and Pop/snap_crackle_and_pop.cpp
#include <iostream>
using namespace std;

int main() {

    int b;
    for (int i = 0; i < 6; i++) {
        cin >> b;
        cout << ((b % 3 == 0) ? "Pop " : ((b % 2 == 0) ? "Crackle " : "Snap "));
    }

    return 0;
}